package.json

```diff
@@ -6,13 +6,13 @@
     "url": "https://github.com/hyperledger/caliper"
   },
   "scripts": {
-    "test": "npm run lint && npm run nyc && npm run e2e",
+    "test": "npm run lint && npm run nyc && npm run e2e -- ",
```

Comment:
Running a benchmark should be independent of running linting and coverage tests. The scripts, in general, need to be organized, so let's start clean with this one.
My suggestions is that a `bench` command should be used that delegates to a common `main.js` file. Something like this:
`"bench": "node ./scripts/main.js"`

So running a benchmark would look like this:
`npm run bench -- -c ./bechmark/simple/config-linear-rate.yaml -n ./network/fabric-v1.1/dev/fabric-node.json`

Reply:
That makes much more sense :) I've added a new bench target